src/mame/vtech/geniusjr.cpp

```diff
@@ -420,6 +432,7 @@ COMP( 1993,  gjmovie,  0,       0,      gjmovie,  geniusjr, geniusjr_state, empt
 COMP( 199?,  pitagjr,  gjmovie, 0,      gjmovie,  geniusjr, geniusjr_state, empty_init, "VTech",   "Pitagorin Junior",                  MACHINE_IS_SKELETON )
 COMP( 1996,  gjrstar,  0,       0,      gjrstar,  geniusjr, geniusjr_state, empty_init, "VTech",   "Genius Junior Redstar (Germany)",   MACHINE_IS_SKELETON )
 COMP( 1996,  gjrstar2, gjrstar, 0,      gjrstar,  geniusjr, geniusjr_state, empty_init, "VTech",   "Genius Junior Redstar 2 (Germany)", MACHINE_IS_SKELETON )
+COMP( 1995,  pcompelr, gjrstar, 0,      gjrstar,  geniusjr, geniusjr_state, empty_init, "VTech",   "Precomputer Electronik (Russia)",   MACHINE_IS_SKELETON )
```

Comment:
Is it actually written like that in Latin script?

Reply:
It's "Прекомпьютер Электроник"

How should I write it?

Reply:
![IMG20230823210022](https://github.com/mamedev/mame/assets/5889052/8632aacf-0aea-42c1-aef5-03fb426ace4c)

Reply:
should probably be Prekomputer Elektronik instead of Precomputer Electronik

Reply:
@cuavas A final recommendation about the transliteration? Thanks!

Reply:
I think I’ll go with “Precomputer” since it’s clearly a transliterated trademark/loanword (native Russian for “computer” is “вычислитель” (vychiclityel').